docs/tempo/website/setup/_index.md

```diff
@@ -1,40 +1,38 @@
 ---
-title: Set up a Tempo cluster
-menuTitle: Set up a Tempo cluster
-description: Learn how to set up a Tempo cluster and visualize data
+title: Set up a Tempo server or cluster
+menuTitle: Set up a Tempo server or cluster
+description: Learn how to set up a Tempo server or cluster and visualize data
 weight: 150
 ---
 
-# Set up a Tempo cluster
+# Set up a Tempo server or cluster
 
 Tempo is available as a pre-compiled binary, a Docker image, and as common OS-specific packaging.
 
-No matter which option you choose, setting up your Tempo cluster involves: 
+You can choose to install Tempo on a single system or as a cluster.
```

Comment:
You can choose to install Tempo as single service or a distributed set of services?

Reply:
This brings up a good point. I don't think we have instructions on using Tempo as a distributed set of services. We have the Tanka setup for a cluster and the linux setup for a single node.

Reply:
```suggestion
This section describes how to set up Tempo on a single Linux node or as a cluster using Kubernetes and Tanka. Tempo can also be set up as a distributed set of services.  
```

Reply:
i like the clarification in your suggestion. i think this plays into the concerns above. "server" and "cluster" are just different words then we use throughout the rest of the docs.